example/ledger-keys/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/binance-chain/go-sdk/common/crypto/ledger"
+	"github.com/binance-chain/go-sdk/keys"
+	"github.com/binance-chain/go-sdk/types"
+	"github.com/binance-chain/go-sdk/types/msg"
+	"github.com/binance-chain/go-sdk/types/tx"
+)
+
+func main() {
+	bip44Params := keys.NewFundraiserParams(0, 0)
+	keyManager1, err := keys.NewLedgerKeyManager(bip44Params.DerivationPath())
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	fmt.Println(fmt.Sprintf("address: %s", keyManager1.GetAddr()))
+	fmt.Println(fmt.Sprintf("pubkey: %s", hex.EncodeToString(keyManager1.GetPrivKey().PubKey().Bytes())))
+
+	receiverAddr, err := types.AccAddressFromBech32("bnb1pkppf2ar3wj38cu7y8khyg0clmhvf2f2nzt4w6")
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	sendMsg := msg.CreateSendMsg(keyManager1.GetAddr(), types.Coins{types.Coin{Denom: "BNB", Amount: 100000000000000}}, []msg.Transfer{{receiverAddr, types.Coins{types.Coin{Denom: "BNB", Amount: 100000000000000}}}})
```

Comment:
What is this? It seems that it should be in a test file, not in `main()`

Reply:
The main purpose I place this code in `main()` is that running this code reqiures there is a connected ledger device. We can't run it like normal unit test.

Reply:
You could skip the test if an env variable is not set.

```
func TestLedger(t *testing.T) {
  if os.Getenv("TEST_LEDGER") != "" {
    t.Skip("Skipping Ledger test, set TEST_LEDGER to run")
  }
}
```

Reply:
You could skip the test if an env variable is not set.

```
func TestLedger(t *testing.T) {
  if os.Getenv("TEST_LEDGER") == "" {
    t.Skip("Skipping Ledger test, set TEST_LEDGER to run")
  }
}
```

Reply:
```
// +build cgo,ledger
```
To support ledger, we have to add above two build tags. Besides, the main purpose I add this example is not to verify the code functionality. It is to tell users how to develop a application with ledger. So think a `main` file is more proper.
@notatestuser 
Do you agree with me?

Reply:
@notatestuser 
Do you have any comment?

Reply:
I can understand this looks more like a test instead of an example. 

If this is for an example, we should make it a real to test in testnet. i.e. derive the address from Ledger key and transfer to him/herself with proper testnet chain id.

Reply:
as far as I know, the build tags are unrelated to the fact that you could use an environment variable to skip the test.

Reply:
I refactor this example. Now it will send a transfer transaction to testnet.